src/main/java/codeu/controller/GeneratingData.java

```diff
@@ -0,0 +1,133 @@
+package codeu.controller;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.UUID;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.mindrot.jbcrypt.BCrypt;
+import org.omg.CORBA.portable.InputStream;
+
+import codeu.model.data.Activity;
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ActivityStore;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+/**
+ * Servlet implementation class GeneratingData
+ */
+public class GeneratingData extends HttpServlet {
+    /** Store class that gives access to Conversations. */
+    private ConversationStore conversationStore;
+    /** Store class that gives access to Messages. */
+    private MessageStore messageStore;
+    /** Store class that gives access to Users. */
+    private UserStore userStore;
+    
+	private String[] users = new String[]{"boss", "manager", "ted", "frank", "linda", "exuser"};
+
+
+    /** Set up state for handling chat requests. */
+    @Override
+    public void init() throws ServletException {
+      super.init();
+      setConversationStore(ConversationStore.getInstance());
+      setMessageStore(MessageStore.getInstance());
+      setUserStore(UserStore.getInstance());
+    }
+    
+    void setConversationStore(ConversationStore conversationStore) {
+        this.conversationStore = conversationStore;
+    }
+    
+    void setMessageStore(MessageStore messageStore) {
+        this.messageStore = messageStore;
+    }
+    
+    void setUserStore(UserStore userStore){
+    	this.userStore = userStore;
+    }
+    
+      
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		ServletContext cntxt = getServletContext();
+		FileInputStream fstream = 
+                new FileInputStream(getServletContext().getRealPath("/datafile"));	
+		
+		//six mock users - will randomly be assigned a message
+		for(String user: users){
+		  newUser(user);
+		}
+		
+		//one convo for now - can expand later
+		String conversation = "codeu_faq";
+
+	    newPrivateConvo("boss", "codeu_faq");
```

Comment:
Use the `conversation` variable you initialized above.

Reply:
Done.